AndelaOSP/Andela-Socials
# #166962667 Add MVP features to develop

#### What Does This PR Do?
- This PR introduces all the implementation and fixes built for the MVP into the develop branch to be deployed to the staging environment
#### Description Of Task To Be Completed
- [x] Make all MVP features and fixes available in the `develop` branch
#### Any Background Context You Want To Provide?
- N/A
#### How can this be manually tested?
- Start the application
- Test to see that all features in the `mvp-develop` branch have been successfully introduced into the `develop` branch
#### What are the relevant pivotal tracker stories?
[#166962667](https://www.pivotaltracker.com/story/show/166962667)
#### Screenshot(s)
